tests/test_serialization.py

```diff
@@ -66,7 +66,12 @@ class MyWebPage(WebPage):
     url = ResponseUrl(url_str)
 
     serialized_deps = serialize([book_list_html_response, url])
-    other_json = f'{{"_encoding": "utf-8", "headers": [], "status": null, "url": "{url_str}"}}'.encode()
+    other_json = f"""{{
+  "_encoding": "utf-8",
+  "headers": [],
+  "status": null,
+  "url": "{url_str}"
+}}""".encode()
```

Comment:
I don't love this test, it was tricky to get it right, and the diffs in case of failure are obscure, because they include complete httpResponseBody content. Not sure how to improve on it, maybe we can leave it as-is for now.